packages/dashboard/src/components/Restore/RestoreFromLocal.tsx

```diff
@@ -77,79 +96,101 @@ export const RestoreFromLocal = memo(() => {
         if (!file) return
 
         try {
-            const decrypted = await decryptBackup(encode(password), await file.arrayBuffer())
-            setBackupValue(JSON.stringify(decode(decrypted)))
+            setReadingFile(true)
+            const [decrypted] = await Promise.all([
```

Comment:
Use `timeout` from `'@masknet/kit'`.

Reply:
No, it's for minimal timeout, but at least waiting time for fake processing.